spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc

```diff
@@ -20,13 +20,19 @@ mvn spring-boot:run
 ----
 
 Browse to `localhost:8080/log`. This will trigger the `ExampleController.log()` method,
-which uses a typical logger supporting Logback to write a line of text to the Stackdriver log. However,
-you will need to specify the trace ID in the `x-cloud-trace-context` header manually to see a trace ID
+which uses a typical logger supporting Logback to write a line of text to the Stackdriver log.
+However, you will need to specify the trace ID in the `x-cloud-trace-context` or `X-B3-TraceId` headers manually to see a trace ID
 label appear in Stackdriver:
 ----
 curl --header "x-cloud-trace-context: [Some-Trace-ID]" localhost:8080/log
 ----
 
+The `spring.cloud.gcp.logging.traceIdExtractor=xcloud_zipkin` setting in `application.properties`
```

Comment:
Should we also add an `application.properties` to the sample with this property set?
This would make it easier for people to modify the sample to use a different header extractor.

Reply:
This is actually the setting in that file right now

Reply:
Oh, sorry, looked in the wrong place.